ML/MLService.cs

```diff
@@ -0,0 +1,208 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.ML;
+using Chatbee.ML.Models;
+using System.Runtime.Serialization.Formatters.Binary;
+using System.IO;
+
+namespace Chatbee.ML
+{
+    public class MLService : IMLService
+    {
+        public DateTime ServiceLoaded { get; set; }
```

Comment:
What is the point of this?

Reply:
The purpose is that the Service will track the datetime that the service itself (which contains the models) was loaded.  This is useful for debugging because if the service reaches end-of-life through disposal, you can check to see when the service was started.